### DISCUSSION TITLE
Bring back the Collection View

### DISCUSSION THREAD

**Discussion by isgapetti** [GitbookIO/community#819] [Feature Requests]
I don't know why you guys removed the ability to click on a Collection. Now I have to keep opening the sidebar.

![image](https://github.com/user-attachments/assets/6fa6dfba-366b-4b5b-98c2-856dcbf84347)

**Comment by ahcsco** [comment]
I sure can say that it gets frustrating with so many frequent UI changes. You get to learn how to do something and poof, it's gone. It's appreciated of course with all the great improvements but the pace sometimes is too fast and sometimes for the worse. I agree, I missed the collections view too.

**Comment by isgapetti** [comment]
Yeah I don't get why they removed it at all. Its completely optional but they chose to remove it.
